app/assets/javascripts/browse_everything/behavior.js.coffee

```diff
@@ -304,7 +304,11 @@ auto_toggle = ->
   triggers = $('*[data-toggle=browse-everything]')
   triggers.each () -> $(this).browseEverything($(this).data())
 
-if Turbolinks?
-  $(document).on 'page:change', auto_toggle
+if Turbolinks && Turbolinks.supported
```

Comment:
Here's what we have in Blacklight:

```
if (typeof Turbolinks !== 'undefined' && Turbolinks.supported) {
```

I'm just not sure if `if Turbolinks` compiles down to that?